morpheus-graphql-core/src/Data/Morpheus/Types/Internal/AST/Error.hs

```diff
@@ -15,7 +15,8 @@ module Data.Morpheus.Types.Internal.AST.Error
     isInternal,
     GQLErrors,
     GQLError
-      ( message,
+      ( GQLError,
```

Comment:
please hide GQLError constructor again, users should use new approach of the error generating.

on the second hand i have flexibility to extend error type how i wish.